env.example

```diff
@@ -0,0 +1,6 @@
+APP_PORT=4000
+
+APP_URL=http://localhost:4000
+
+GITHUB_ACCOUNT_LOGIN=yourGithubLogin
```

Comment:
I suggest to name `AUTH_USERNAME` and add a comment, it should be your GitHub username.

I also suggest to group `APP_URL`, `AUTH_USERNAME` and `AUTH_PASSWORD` together with a comment, they are needed to make the script work.

Reply:
Done